homeassistant/components/sensor/miflora.py

```diff
@@ -31,9 +29,7 @@
 DEFAULT_NAME = 'Mi Flora'
 DEFAULT_RETRIES = 2
 DEFAULT_TIMEOUT = 10
-
-UPDATE_INTERVAL = 1200
-MIN_TIME_BETWEEN_UPDATES = timedelta(seconds=UPDATE_INTERVAL)
```

Comment:
To elaborate a little more, Home Assistant uses the module defined `SCAN_INTERVAL` as the default poll interval, and it can be overwritten by the user by defining `scan_interval` in their configuration for the platform. You should be able to eliminate `CONF_CACHE` entirely.